app/main.py

```diff
@@ -2,24 +2,61 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: object, owner: type) -> None:
+        getattr(instance, self.protected_name)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if not self.min_amount <= value <= self.max_amount:
+            raise ValueError("You can not use the slide :( ")
```

Comment:
Sad. But, maybe, since it's class IntegerRange we could add parameters, like, idk:
```suggestion
            raise ValueError("f"Value {value} is out of allowed range [{self.min_amount}, {self.max_amount}]"")
```